### PR TITLE
New package: jmtpfs-0.5

### PR DESCRIPTION
**Pull request by jorgenatz**
This is a utility for mounting mtp devices as FUSE filesystems.

I am not sure of how reliable a maintainer I would be. Should I designate a common maintainer with their permission, put myself anyway, or avoid submitting the package?

Thank you for any reply, and I apologize for anything I missed.

### CODE REVIEW

**Comment by jorgenatz** [comment]
Thank you for giving your time to check my template.

I apologize if I do not completely understand the SPDX licensing taxonomy, but it seems to me, that according to this issue,:
[https://github.com/spdx/license-list-XML/issues/617](https://github.com/spdx/license-list-XML/issues/617)
that the license would be described by "GPL-3.0-or-later" by virtue of including the phrase "or (at your option) any later version" in the recommended grant wording.

File: srcpkgs/jmtpfs/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'jmtpfs'
+pkgname=jmtpfs
+version=0.5
+revision=1
+build_style=gnu-configure
+hostmakedepends="git pkg-config"
+makedepends="libmtp-devel fuse-devel file-devel"
+short_desc="FUSE-based filesystem for accessing MTP devices"
+maintainer=???
+license="GPL-3.0-or-later"
```
[the remark this answers is not in the document]
Comment (jorgenatz) [reply srcpkgs/jmtpfs/template]: I apologize if I do not completely understand the SPDX licensing taxonomy, but it seems to me, that according to this issue,:
[spdx/license-list-XML#617](https://github.com/spdx/license-list-XML/issues/617)
that the license would be described by "GPL-3.0-or-later" by virtue of including the phrase "or (at your option) any later version" in the recommended grant wording.

[the remark this answers is not in the document]
Comment (Hoshpak) [reply srcpkgs/jmtpfs/template]: If that is the case, it is correct. However I did not find the phrase included anywhere. I looked at some random source files and found (e.g. in TemporaryFile.cpp):

```
 * This software is free software; you can redistribute it and/or
 * modify it under the terms of the GNU General Public License
 * version 3 as published by the Free Software Foundation.
```
which does not specifiy the later version.

[the remark this answers is not in the document]
Comment (jorgenatz) [reply srcpkgs/jmtpfs/template]: Ah, I did not see those. Thanks for pointing those out. I had incorrectly looked in the COPYING file for the "any later version" phrase. Unfortunately I am not able to amend the commits myself (linux machine broke). Feel free to change my pull request.

**Comment by Hoshpak** [comment]
amended and merged manually.

**Pull request closed without merging**